<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateForum</class>
 <widget class="QDialog" name="CreateForum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create new Forum</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/images/rstray3.png</normaloff>:/images/rstray3.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="HeaderFrame" name="headerFrame"/>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="forumName"/>
        </item>
       </layout>
      </item>
      <item row="0" column="1" rowspan="5">
       <widget class="QDockWidget" name="contactsdockWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>524287</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>220</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>check peers you would like to share private publish key with</string>
        </property>
        <property name="floating">
         <bool>false</bool>
        </property>
        <property name="features">
         <set>QDockWidget::NoDockWidgetFeatures</set>
        </property>
        <property name="windowTitle">
         <string>Share Key With</string>
        </property>
        <widget class="QWidget" name="dockWidgetContents">
         <layout class="QGridLayout" name="_2">
          <property name="margin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="FriendSelectionWidget" name="keyShareList" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>4</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>20</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>220</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="forumDesc"/>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="tyoeGroupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Type:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="margin">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <widget class="QRadioButton" name="typePublic">
           <property name="text">
            <string>Public - Anyone can read and publish (Shared Publish Key)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="typePrivate">
           <property name="text">
            <string>Restricted - Anyone can read, limited publishing (Private Publish Key)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="typeEncrypted">
           <property name="text">
            <string>Private - (Private Publish Key required to view Messages)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="msgGroupBox_2">
        <property name="title">
         <string>Key Sharing</string>
        </property>
        <layout class="QVBoxLayout" name="_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
         <item>
          <widget class="QCheckBox" name="pubKeyShare_cb">
           <property name="toolTip">
            <string>Key recipients can publish to restricted-type channels, and can view and publish  for private-type channels</string>
           </property>
           <property name="text">
            <string>Share Private Publish Key</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QGroupBox" name="msgGroupBox">
        <property name="title">
         <string>Allowed Messages</string>
        </property>
        <layout class="QVBoxLayout" name="_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>6</number>
         </property>
         <item>
          <widget class="QRadioButton" name="msgAuth">
           <property name="text">
            <string>Authenticated Messages</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="msgAnon">
           <property name="text">
            <string>Anonymous Messages</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HeaderFrame</class>
   <extends>QFrame</extends>
   <header>gui/common/HeaderFrame.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FriendSelectionWidget</class>
   <extends>QWidget</extends>
   <header>gui/common/FriendSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
